Git_Tutorial.md

```diff
@@ -0,0 +1,168 @@
+# Version Control System using Git
+
+#### What is Git?
+
+-----
+
+**Git** is  a distributed version-control system for tracking changes in source code during software development. 
```

Comment:
Can we add something more simpler for newbies to understand what git is. Like a video or something ?